packages/webapp/src/components/Form/ContextForm/WithStepperProgressBar.tsx

```diff
@@ -108,17 +118,26 @@ export const WithStepperProgressBar = ({
     } catch (e) {
       console.error(`Error during canceling ${cancelModalTitle}: ${e}`);
     }
+    setIsEditing?.(false);
+  };
+
+  const handleDismissModal = () => {
+    setTransition({ unblock: undefined, retry: undefined });
+    setShowCancelFlow?.(false);
+    setIsEditing?.(true);
```

Comment:
Are there any situations where `handleDismissModal` is called while `isEditing` is `false`?

Reply:
Ah no, thank you. That is a holdover from a previously implementation of `onCancel()` that flipped the boolean first but I don't Anto's implementation of `onCancel` could ever get to that state; I'll remove it! 🙏